charts/dbildungscloud-iam-keycloak/templates/deployment.yaml

```diff
@@ -1,102 +1,101 @@
 apiVersion: apps/v1
```

Comment:
Please add all Security relevant inputs. The helm scanner wants e.g. :  CPU and Memory Limits and Requests. NET_RAW Capabilities Not Being Dropped. Seccomp Profile Is Not Configured. Service Account Token Automount Not Disabled.